### PR TITLE
Fix flaky test for dataSize RecordsQuery

### PR DESCRIPTION
**Pull request by diehuxx**
A test for `RecordsQuery` was flaky since the response array was not meant to be sorted, but the test depended on sort order. Updating the test to check that the unordered arrays match.

### CODE REVIEW

File: tests/handlers/records-query.spec.ts [hunk 1]
```diff
@@ -221,7 +221,13 @@ export function testRecordsQueryHandler(): void {
         const reply1 = await dwn.processMessage(alice.did, recordsQuery1.message);
         expect(reply1.entries?.length).to.equal(2);
         expect(reply1.entries![0].encodedData).to.equal(Encoder.bytesToBase64Url(write2.dataBytes!));
```
Comment (LiranCohen) [inline tests/handlers/records-query.spec.ts:223]:
```suggestion
```

**Review by LiranCohen — APPROVED** [review]
Nice! Just missed one of the lines.

**Review by thehenrytsai — APPROVED** [review]
🙏

**Review by LiranCohen — APPROVED** [review]

**Review by thehenrytsai — APPROVED** [review]
3rd time lucky.

**Pull request merged**